VCFv4.3.tex

```diff
@@ -503,41 +503,100 @@ \subsection{VCF tag naming conventions}
 ##INFO=<ID=END,Number=1,Type=Integer,Description="End position of the variant described in this record">
 \end{verbatim}
 \normalsize
-For precise variants, END is POS + length of REF allele - 1, and the for imprecise variants the corresponding best estimate.
+For precise variants, END is POS + length of REF allele - 1.
 
 \footnotesize
 \begin{verbatim}
 ##INFO=<ID=SVTYPE,Number=1,Type=String,Description="Type of structural variant">
 \end{verbatim}
 \normalsize
-This key can be derived from the REF/ALT fields but is useful for filtering. The reserved values must be used for the types listed below:
+SVTYPE is required for all structural variants. The following reserved values should be used whenever possible; indented values are subtypes that should be used if evidence supports them (see examples in Section 5.3, Encoding Structural Variants):\vspace{0.3cm}\\
 \begin{itemize}
-  \item DEL: Deletion relative to the reference
-  \item INS: Insertion of novel sequence relative to the reference
-  \item DUP: Region of elevated copy number relative to the reference
-  \item INV: Inversion of reference sequence
-  \item CNV: Copy number variable region (may be both deletion and duplication)
+\item [\null] INS:  Insertion relative to the reference
+	\begin{itemize}
+	\item [\null] MEI:  Mobile element insertion relative to the reference
+		\begin{itemize}
+		\item [\null] ALU:  Alu mobile element insertion relative to the reference
+		\item [\null] LINE1:  LINE1 mobile element insertion relative to the reference
+		\item [\null] SVA:  SVA mobile element insertion relative to the reference
+		\item [\null] HERV:  HERV mobile element insertion relative to the reference
+		\end{itemize}
+	\item [\null] RET:  Retrotransposon insertion relative to the reference
+	\end{itemize}
+\item [\null] DEL:  Deletion relative to the reference
+	\begin{itemize}
+	\item [\null] -MEI:  Absence of a mobile element present in the reference
+		\begin{itemize}
+		\item [\null] -ALU:  Absence of a Alu mobile element present in the reference
+		\item [\null] -LINE1:  Absence of a LINE1 mobile element present in the reference
+		\item [\null] -SVA:  Absence of a SVA mobile element present in the reference
+		\item [\null] -HERV:  Absence of a HERV mobile element present in the reference
+		\end{itemize}
+	\item [\null] -RET:  Absence of a retrotransposon present in the reference
+	\end{itemize}
+\item [\null] TRA:  Translocation
+\item [\null] STR:  Short tandem repeat variation
+\item [\null] INV:  Inversion relative to the reference
+\item [\null] DUP:  Duplication
+	\begin{itemize}
+	\item [\null] TANDUP:  Tandem duplication (one of Direct, Inverted-upstream, or Inverted-downstream)
+	\item [\null] DISDUP:  Dispersed duplication
+	\end{itemize}
+\item [\null] DELINS:  Deletion-insertion (replacement of reference sequence with sequence of different content or length)
+\item [\null] OTHER:  Complex variant not covered elsewhere
```

Comment:
I agree that a symbolic alt allele for complex event has its value, because being easily interpretable is a virtue, as long as we have a healthy suspicion on the provided interpretation, which is what the subtypes proposed here are trying to do.

On the other hand, this proposal of `OTHER` is too ambiguous. It has almost no power of restraining what can be put here. It should at least be mentioned that for a record to be classified as "complex", at least two types of events each would be associated with a distinct 1st level alt alle must be involved.

And a third point I would like to make: adding these subtypes is "unfair" to the other imaginable dozens (if not hundreds) subtypes.

Now regarding @d-cameron 's comments about equivalent BND records connected with their `EVENT` annotation, I agree there should be a tool providing such functionality. But, IMHO, for complex variant, what we should do is to signal that a series of simple event is connected together is a non-trivial way so a region of the reference is rearranged. A caller should output (best estimate of) this region's boundary in a as compact as possible fashion, and preferably output the reference and alternative haplotypes. This would fulfill the following needs:
* the affected genomic locations are specified, this is beneficial as compared to a series of BND records which must be parsed all together to clearly define the boundaries that cluster these events together;
* event could be easily interpreted (though one should not blindly trust the interpretation);
* the concrete sequence is available for downstream analysis (genotyping, re-interpretation) and these downstream analysis doesn't need to painstakingly reconstruct the haplotype.